src/App.tsx

```diff
@@ -1,24 +1,137 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { createTodo, deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Error } from './components/Error';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { useSort } from './utils/useSort';
+import { Sort } from './types/Sort';
 
-const USER_ID = 0;
+const USER_ID = 6771;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [errorType, setErrorType] = useState<string>('');
```

Comment:
there is no need to use 2 state variables for error
```suggestion
  const [errorType, setErrorType] = useState<string>('');
  const isError = !!errorType || errorType !== '';
```